tests/pytests/unit/returners/test_local_cache.py

```diff
@@ -0,0 +1,16 @@
+from salt.returners.local_cache import _remove_job_dir
+from tests.support.mock import patch
+
+
+def test_remove_job_dir():
```

Comment:
```suggestion
@pytest.mark.parametrize("exc", (NotADirectoryError, OSError))
def test_remove_job_dir(exc):
```
Instead of iterating on the test.